console/src/main/scala/io/joern/console/cpgcreation/package.scala

```diff
@@ -112,11 +115,11 @@ package object cpgcreation {
     }
   }
 
-  def withFileInTmpFile(inputPath: String)(f: File => Try[String]): Try[String] = {
-    val dir = File.newTemporaryDirectory("cpgcreation")
-    File(inputPath).copyToDirectory(dir)
+  def withFileInTmpFile(inputPath: String)(f: JFile => Try[String]): Try[String] = {
```

Comment:
Maybe consider aliasing `better.files.File` as `BetterFile` and keeping the java one as `File`